doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -589,6 +594,18 @@ The Fast Pair GATT service modifies default values of related Kconfig options to
 The service also enables the needed functionalities using Kconfig select statement.
 For details, see the :ref:`bt_fast_pair_readme` Bluetooth service documentation in the |NCS|.
 
+.. _ug_bt_fast_pair_gatt_service_subsequent_pairing:
+
+Subsequent Pairing
+==================
+
+The Fast Pair specification supports the Subsequent Pairing feature.
+Subsequent pairing is the later pairing between a Fast Pair Provider that was initial-paired to your Google account and your other Fast Pair Seeker where the same Google Account is logged-in.
+
+You can enable the :kconfig:option:`CONFIG_BT_FAST_PAIR_SUBSEQUENT_PAIRING` configuration option to support the Subsequent Pairing feature during the `Fast Pair Procedure`_.
+
+As a consequence, the Fast Pair not discoverable advertising with UI indications is only available when the Subsequent Pairing feature is supported, as this advertising type is used to trigger the subsequent pairing UI flow.
```

Comment:
```suggestion
The Fast Pair specification supports the subsequent pairing feature.
Subsequent pairing refers to the connection between a Fast Pair Provider, initially paired with your Google account, and another Fast Pair Seeker logged into the same account.

To support the subsequent pairing feature in the `Fast Pair Procedure`_, enable the :kconfig:option:`CONFIG_BT_FAST_PAIR_SUBSEQUENT_PAIRING` Kconfig option.

Consequently, the Fast Pair not discoverable advertising with UI indications, which is used to trigger the subsequent pairing UI flow, is only available when the subsequent pairing feature is supported. 
```
tried to make it flow better.
Used lowercase for subsequent pairing - let me know if it was agreed it should be capitalized
Also, removed "the later" / the latter from the 2nd sentence, not sure if it's necessary

Reply:
We also use lower-case approach in the SML tables so I won't use capitalization:

https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/software_maturity.html

Reply:
`Subsequent pairing refers to the procedure between a Fast Pair Provider, initially paired with your Google account, and another Fast Pair Seeker logged into the same account.`

Cannot really say connection as it is procedure. This definition was also taken from the Google specification:

https://developers.google.com/nearby/fast-pair/fast-pair-certification-guideline

Reply:
"Initial pairing" -> former pairing
"Subsequent pairing" -> later pairing

Reply:
the current edit looks good - thanks! :)